x-pack/test/observability_solution_api_integration/README.md

```diff
@@ -0,0 +1,80 @@
+# observability_solution_api_integration
+
+This directory serves as a centralized location to place the observability solution tests that run in Serverless and ESS environments
+
+## Subdirectories
+
+1. `config` stores base configurations specific to both the Serverless and ESS environments. These configurations build upon the base configuration provided by `x-pack/test_serverless` and `x-pack/test/api_integration`, incorporating additional settings such as environmental variables and tagging options
+
+2. `test_suites` directory houses all the tests along with the utility functions.
+
+## Overview
+
+- In this directory suiteTags is utilized to assign tags to specific test suites. This tagging system enables the ability to selectively apply tags to test suites, facilitating the exclusion of specific test suite as needed.
+
+- Test suites are prefixed with specific tags to determine their execution in particular environments or to exclude them from specific environments.
+
+- We are using the following tags:
+    * `@ess`: Runs in an ESS environment (on-prem installation) as part of the CI validation on PRs.
```

Comment:
@pheyos Let's agree on the tags we want to use.

Reply:
no need to apply tags, at least at the moment. We have 2 config files:
- x-pack/test/api_integration/deployment_agnostic/oblt.stateful.config.ts
- x-pack/test/api_integration/deployment_agnostic/oblt.serverless.config.ts

and index file to load all Oblt tests:
- x-pack/test/api_integration/deployment_agnostic/oblt.index.ts

If the test is added to `deployment_agnostic/` folder, it is assumed by default to be run both in serverless and stateful without extra filtering